3_Open_Source_Software.md

```diff
@@ -90,6 +91,8 @@ The differences amongst the various licences in the popular GNU suite illustrate
 
 (table?)
 
+Taking all of the above into account by default the government will follow a similar policy to that which is outlined by the GNU Public License version 3 Section 5. Namely that projects published by the Government of Canada are to be freely available for modification and re-distribution by Canadians, Canadian companies, or countries that enter into an agreement with Canada, on the condition that their modifications are licensed under the same agreement. The intent of said stance is to foster the Canadian entrepreneurship and acknowledges that all work done, and code produced, is done for the benefit of Canadians. 
```

Comment:
This should probably be in the Open Source Code section.  Also I don't know what the final recommendation will be, but permissive licenses should also be considered. See #4 

I would like to see a balanced approach where reciprocal is selected for user tools and permissive for frameworks and libraries..